tso-server/src/main/java/org/apache/omid/tso/ReplyProcessorImpl.java

```diff
@@ -67,7 +69,7 @@
     private final Meter timestampMeter;
 
     @Inject
-    ReplyProcessorImpl(MetricsRegistry metrics, Panicker panicker, ObjectPool<Batch> batchPool) {
+    ReplyProcessorImpl(TSOServerConfig config, MetricsRegistry metrics, Panicker panicker, ObjectPool<Batch> batchPool) {
```

Comment:
Same comment as above. Inject strategy instead of tso config. Should be:

```java
    @Inject
    ReplyProcessorImpl(@Named("ReplyStrategy") WaitStrategy strategy,
                       MetricsRegistry metrics,
                       Panicker panicker, 
                       ObjectPool<Batch> batchPool) {

        // ------------------------------------------------------------------------------------------------------------
        // Disruptor initialization
        // ------------------------------------------------------------------------------------------------------------

        ThreadFactoryBuilder threadFactory = new ThreadFactoryBuilder().setNameFormat("reply-%d");
        this.disruptorExec = Executors.newSingleThreadExecutor(threadFactory.build());

        this.disruptor = new Disruptor<>(EVENT_FACTORY, 1 << 12, disruptorExec, MULTI, strategy);
...
```